README.md

```diff
@@ -1,3 +1,158 @@
+
+Loading this module through a script tag will make the `fdp` object available in the global namespace.
+
+```html
+<script src="https://unpkg.com/@fairdatasociety/fair-data-protocol/dist/index.browser.min.js"></script>
+```
+
+## Usage
+
+Creating FDP account
+
+```js
+import { FairDataProtocol } from '@fairdatasociety/fair-data-protocol'
+
+const fdp = new FairDataProtocol('http://localhost:1633', 'http://localhost:1635')
+
+const account = await fdp.account.register('myusername', 'mypassword')
+console.log(account) // prints account information
+```
+
+Login with FDP account
+
+```js
+// before login, we need to enter the username of the account and the Ethereum address associated with it
+await fdp.account.setUserAddress('otherusername', '0x.....')
+const wallet = await fdp.account.login('otherusername', 'mypassword')
+console.log(wallet) // prints downloaded and decrypted wallet
+```
+
+Getting list of pods
+
+```js
+const pods = await fdp.personalStorage.list()
+console.log(pods) // prints list of user's pods
+```
+
+## Documentation
+
+You can generate API docs locally with:
+
+```sh
+npm run docs
+```
+
+The generated docs can be viewed in browser by opening `./docs/index.html`
+
+## Contribute
+
+There are some ways you can make this module better:
+
+- Consult our [open issues](https://github.com/fairDataSociety/fair-data-protocol/issues) and take on one of them
+- Help our tests reach 100% coverage!
+
+### Setup
+
+Install project dependencies with
+
+```sh
+npm i
+```
+
+### Test
+
+The tests run in both context: node and dom with Jest.
+
+To run the integration tests, you need to use our [`bee-factory`](https://github.com/fairDataSociety/bee-factory/) project. Clone the repo, you can use our prebuilt Docker images with setting env. variables like:
+
+```bash
+export FAIROS_VERSION="0.7.3"
+export BLOCKCHAIN_VERSION="1.2.0"
+export BEE_ENV_PREFIX="swarm-test"
+export BEE_IMAGE_PREFIX="docker.pkg.github.com/ethersphere/bee-factory"
+```
+
+Customize these values based on which FairOS version you want to run. After the env. variables are set use the `./scripts/environment.sh` script with `start --fairos` parameter.
+
+If you want to skip creation of postage stamps every run of integration tests you can create stamps for both nodes and set them under env. variables `BEE_POSTAGE` and `BEE_PEER_POSTAGE`.
+
+By default, for integration tests two bee nodes are expected to run on localhost on addresses `http://localhost:1633` and `http://localhost:1635`. These are the default values for the `bee-factory` script.
+If you want to use custom setup, you can change the behavior of tests to different addresses using environment variables `BEE_API_URL`, `BEE_DEBUG_API_URL`, `BEE_PEER_DEBUG_API_URL` and `BEE_PEER_API_URL`.
+
+In Visual Studio environment, the tests have been set up to run against your local bee node on `http://localhost:1633`
+To run Jest tests, choose the `vscode-jest-tests` CI job under the Run tab.
```

Comment:
The configuration files are not defined for this.
Anyway, I would remove this section, we don't need it